src/async-wrap.h

```diff
@@ -87,13 +87,22 @@ class AsyncWrap : public BaseObject {
     PROVIDERS_LENGTH,
   };
 
+  enum Flags {
```

Comment:
Wrote it this way to be consistent with StreamBase::Flags. Converting these to `enum class ...` is a bit more involved than I'd like to get here. If we want to convert those, then we can do both AsyncWrap::Flags and StreamBase::Flags at the same time in a separate PR.